src/openforms/emails/tasks.py

```diff
@@ -20,32 +18,24 @@ def send_email_digest() -> None:
     if not (recipients := config.recipients_email_digest):
         return
 
-    period_start = timezone.now() - timedelta(days=1)
+    yesterday = timezone.now() - timedelta(days=1)
 
-    logs = TimelineLogProxy.objects.filter(
-        timestamp__gt=period_start,
-        extra_data__status=Message.STATUS_FAILED,
-        extra_data__include_in_daily_digest=True,
-    ).distinct("content_type", "extra_data__status", "extra_data__event")
+    failed_emails = collect_failed_emails(yesterday)
+    failed_registrations = collect_registrations_failures(yesterday)
 
-    if not logs:
+    if not (failed_emails or failed_registrations):
         return
```

Comment:
Might be worth sending the email even if they are no failures? In that case, this early return should be removed, and I think my comment in the `src/openforms/emails/templates/emails/admin_digest.html` file would make more sense.